packages/legend-studio/src/components/editor/edit-panel/connection-editor/DatabaseBuilderEditor.tsx

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import Dialog from '@material-ui/core/Dialog';
+import type { TreeNodeContainerProps } from '@finos/legend-studio-components';
+import {
+  PanelLoadingIndicator,
+  SaveIcon,
+  FireIcon,
+  clsx,
+  TreeView,
+} from '@finos/legend-studio-components';
+import { ReflexContainer, ReflexElement, ReflexSplitter } from 'react-reflex';
+import { useEffect } from 'react';
+import type {
+  DatabaseBuilderState,
+  DatabaseBuilderTreeData,
+  DatabaseBuilderTreeNodeData,
+} from '../../../../stores/editor-state/element-editor-state/connection/DatabaseBuilderState';
+import {
+  ColumnDatabaseBuilderTreeNodeData,
+  SchemaDatabaseBuilderTreeNodeData,
+  TableDatabaseBuilderTreeNodeData,
+} from '../../../../stores/editor-state/element-editor-state/connection/DatabaseBuilderState';
+import {
```

Comment:
Let's bring these from `@finos/legend-studio-components`

Reply:
not yet. not real use cases for them yet.